pkg/loader/compose/compose.go

```diff
@@ -374,7 +374,10 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 		serviceConfig.Tty = composeServiceConfig.Tty
 		serviceConfig.MemLimit = composeServiceConfig.MemLimit
 		serviceConfig.TmpFs = composeServiceConfig.Tmpfs
-		komposeObject.ServiceConfigs[name] = serviceConfig
+		komposeObject.ServiceConfigs[normalizeServiceNames(name)] = serviceConfig
+		if !reflect.DeepEqual(komposeObject.ServiceConfigs[normalizeServiceNames(name)], komposeObject.ServiceConfigs[name]) {
```

Comment:
@cdrage Test are already there, the function `normalizeServiceNames()` was written by @surajssd but we never used it.
[Tests](https://github.com/kubernetes-incubator/kompose/blob/master/pkg/loader/compose/compose_test.go#L271) for `normalizeServiceNames()`.